conduit/scripts/localwrap

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python
+# Copyright 2019 The Johns Hopkins University Applied Physics Laboratory
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import argparse
+import shutil
+import os
+import subprocess
+import parse
+import resource 
+# Inputs will be of the form
+# TO -> --bucket test_bucket
+#       --directory 12f2/stepname
+#       f1 f2 f3 f4
+# Should upload f1..4 to S3://test_bucket/12f2/stepname
+# FROM -> --bucket test_bucket
+#         --directory 12f2/stepname
+#         f1 f2 f3 f4
+# Should download S3://test_bucket/12f2/stepname/f1..4 to current dir
+
+
+parser = argparse.ArgumentParser(
+)
+# TODO fix dependence on last file being in the same bucket
+# As an implementation note, this will fail if the last download file is from another bucket
+
+parser.add_argument('--input', help='Comma delimited list of files to download in the form /directory/file')
+parser.add_argument('--output', help='Comma delimited list of files to upload in the form /directory/file')
+parser.add_argument('--wf', help = 'Workflow stepname in the form of workflow_name/step_name')
+parser.add_argument('command', nargs=argparse.REMAINDER, help='The actual command being run form of command args ...')
+
+args = parser.parse_args()
+if args.wf:
+    input_path = '../volumes/data/local/' + args.wf.split('/')[0] + '/' 
+    output_path = '../volumes/data/local/' + args.wf + '/' 
+if not os.path.exists(output_path):
+        os.makedirs(output_path) # creates the volume path if it doesn't exist already
+
+if args.input:
+    infiles = args.input.split(',') #list of input files with parent directorys e.g. ['boss_pull_raw.0/pull_output.npy']
+    for f in infiles:
+        fs = f.split('/') #splits directory from file name e.g. ['boss_pull_raw.0', 'pull_output.npy']
```

Comment:
has this been tested on windows? might want to open a ticket for that...

Reply:
#28

Reply:
I will make the changes Raph suggested to decrease the likelihood of it breaking. 

I haven't tested it extensively but it should currently work on Windows since the localwrap code is executed within the docker container. Theoretically as long as the volume is mounted correctly (meaning that the user replaces any drives of the format C:\ with the new format //c/) it should still work fine.